integrations/crowdnode/src/main/res/values/strings-crowdnode.xml

```diff
@@ -144,7 +144,7 @@
     <string name="crowdnode_withdraw_without_limits_online">Withdraw without limits with an online account on CrowdNode website.</string>
     <string name="crowdnode_withdrawal_limits_error">This error is most likely due to exceeding CrowdNode withdrawal limits. Try again later.</string>
     <string name="crowdnode_withdrawal_limits_per_block_title">Please wait before initiating the next withdrawal</string>
-    <string name="crowdnode_withdrawal_limits_per_block_message">Please wait 5 minutes before initiating another withdrawal.</string>
+    <string name="crowdnode_withdrawal_limits_per_block_message">You need to wait 5 minutes before initiating another withdrawal</string>
```

Comment:
This should match https://github.com/dashpay/dash-wallet/pull/1231 as it was copied and pasted into this PR on the one text item.